template/content/server.js

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env node
+'use strict';
+
+var fileStart = Date.now();
+
+var hostname = require('os').hostname;
+var process = require('process');
+var path = require('path');
+var getRepoInfo = require('git-repo-info');
+var parseArgs = require('minimist');
+
+var Application = require('./app.js');
+
+module.exports = main;
+
+/*istanbul ignore else: dead branch*/
+if (require.main === module) {
+    main({
+        argv: parseArgs(process.argv.slice(2))
+    });
+}
+
+function main(opts) {
+    var mainStart = Date.now();
+
+    /*eslint no-process-exit: 0*/
+    var app = Application();
+
+    // attach before throwing exception
+    process.on('uncaughtException', app.clients.uncaught);
+
+    var gitRepo = path.join(__dirname, '.git');
+    var gitSha = getRepoInfo(gitRepo).sha;
+
+    app.bootstrap(onAppReady);
+
+    var bootstrapEnd = Date.now();
+
+    function onAppReady(err) {
+        /*istanbul ignore if: should never happen in prod*/
+        if (err) {
+            app.clients.logger.fatal('Could not start', {
+                error: err
+            });
+            return process.exit(1);
+        }
+
+        var processTitle = opts.argv && opts.argv.processTitle;
```

Comment:
I don't think you should wait until this point to set the process title. You want to do it as one of the first things as process monitoring often depends on process title.